dot/network/service.go

```diff
@@ -352,6 +358,7 @@ func (s *Service) updateMetrics() {
 		case <-s.ctx.Done():
 			return
 		case <-ticker.C:
+			processStartTimeGauge.Set(float64(time.Now().Unix()))
```

Comment:
Is there any way to send it only once? since it's called "process start time" we are not going to have the right value if we are sending it in every tick

Reply:
I agree, this gauge should be set on startup.

Reply:
Good point, I've moved the processStartTimeGauge.Set to inside the service's start method.